app_flutter/lib/commit_box.dart

```diff
@@ -70,15 +60,15 @@ class CommitOverlayContents extends StatelessWidget {
   CommitOverlayContents({
     Key key,
     @required this.parentContext,
-    @required this.widget,
+    @required this.commit,
     @required this.closeCallback,
   }) : super(key: key);
 
   /// The parent context that has the size of the whole screen
   final BuildContext parentContext;
 
-  /// The parent widget that contains state variables
-  final CommitBox widget;
+  /// The commit of the parent widget to show information from
```

Comment:
This doc string is a bit hard to parse.

Maybe `The commit data to display in the overlay.`?

Reply:
Thank you. That is much clearer and the meaning I wanted to convey